msal/src/main/java/com/microsoft/identity/client/controllers/MSALTokenOperation.java

```diff
@@ -0,0 +1,10 @@
+package com.microsoft.identity.client.controllers;
```

Comment:
License is needed at the top of file, also could you please run the auto formatter on this file.

Reply:
Done.